change/@fluentui-react-button-2020-09-22-12-43-20-xgao-fix-theme-package-version.json

```diff
@@ -0,0 +1,8 @@
+{
+  "type": "none",
```

Comment:
I think these should be patch?

Reply:
these `none` packages will be bumped because it's dependency is bumped. i double checked locally using `beachball bump`

Reply:
actually, i should not just think of this from the bumping perspective. this is a rare case but thinking a bit more, having patch feels more correct